Source/Blazorise/wwwroot/fileEdit.js

```diff
@@ -39,8 +40,17 @@ export function removeFile(element, elementId, fileId) {
 }
 
 export function destroy(element, elementId) {
-    var instances = _instances || {};
-    delete instances[elementId];
+    const instances = _instances || {};
+
+    const instance = instances[elementId];
+
+    if (instance) {
+        if (element) {
```

Comment:
Is the check necessary here? It does the same in removeAllFileEntries once again.

Reply:
It is not necessary. But an extra check doesn't hurt.